### ISSUE TITLE
Self host fonts

### ISSUE THREAD

**Issue by XhmikosR** [jakearchibald/svgomg#280]

**Comment by TrySound** [comment]
Why?

**Comment by XhmikosR** [comment]
https://github.com/jakearchibald/svgomg/pull/250#issuecomment-916974688

**Comment by mvoloskov** [comment]
Using system font stack would be a [great solution](https://mrmrs.cc/writing/webfonts/). And it looks good, too!

<img width="784" alt="image" src="https://user-images.githubusercontent.com/14220138/135074972-d157b41e-f535-4698-bedf-4e07e474c187.png">

This is the stack I applied:
```css
	font-family:
		-apple-system,
		BlinkMacSystemFont,
		"Segoe UI",
		Roboto,
		Oxygen,
		Ubuntu,
		Cantarell,
		"Fira Sans",
		"Droid Sans",
		"Helvetica Neue",
		sans-serif,
		"Apple Color Emoji",
		"Segoe UI Emoji",
		"Segoe UI Symbol";
```

**Comment by XhmikosR** [comment]
Yup, agreed :) I'm going to make a PR later so that we test things out with a preview.

**Comment by mvoloskov** [comment]
@XhmikosR I hope [my PR](https://github.com/jakearchibald/svgomg/pull/331) helps :)

**Comment by XhmikosR** [comment]
@mvoloskov thanks for the PR!

That being said, there are a lot more leftovers. I will make a PR myself later after we get the OK from @jakearchibald :)
